iroh-util/src/lib.rs

```diff
@@ -44,15 +44,43 @@ pub async fn block_until_sigint() {
     ctrlc_oneshot.await.unwrap();
 }
 
-/// Path to the iroh home directory.
-pub fn iroh_home_root() -> Option<PathBuf> {
-    let home = home_dir()?;
-    Some(Path::new(&home).join(IROH_DIR))
+/// Returns the path to the user's iroh config directory.
+///
+/// The returned value depends on the operating system and is either a `Some`, containing a value from the following table, or a `None`.
```

Comment:
do our comments use a ruler? on other projects I've line-wrapped at 80 characters of width.

Reply:
yes, cargo fmt checks line length and I think breaks them at 100